components/ParameterForm.vue

```diff
@@ -1,9 +1,11 @@
 <template>
-  <div>
+  <div v-show="pageMounted">
```

Comment:
Luckily, this problem disappears [next PR](https://github.com/jameel-institute/daedalus-web-app/pull/27), because by then we can wait for version data as our signal that everything is mounted.